Report about learning Metabolomics class

Wannapat Masangwunthong 665070039-7

Metabolomics is the systematic study of all the metabolites (<1000 Da) present within a biological system.
A subset of metabolomics is defined as the quantitative measurement of metabolites. The study is specific to the biological sample that in the response to the stimuli such as anticancer agents, environment and genetic modification.

Analytical platforms in metabolomics research
1.Mass spectrometry (MS)
advantage
- High Sensitivity
- Low Amount of Sample
- High Throughput : enabling the analysis of a large number of samples in a relatively short period.
- Metabolite Identification: MS can provide valuable information about the structure and identity of metabolites through techniques such as tandem MS (MS/MS) or accurate mass measurements.
2.Nuclear magnetic resonance (NMR) spectroscopy
advantage
- Easy Sample Preparation
- High Reproducibility: allowing for reliable comparisons of metabolite profiles across different samples or experimental conditions.
- Nondestructive: the samples can be retained for further analysis or validation after NMR measurements.
- Structure Elucidation: NMR spectroscopy provides detailed information about molecular structure, including connectivity, chemical shifts, and coupling patterns, facilitating the identification and characterization of metabolites.

Metabolites consist - Endogenous metabolites : involved in various essential biological functions, such as energy production, cell signaling, and maintenance of cellular homeostasis.
                    - Dietarycomponents : include nutrients (e.g., carbohydrates, proteins, fats), vitamins, minerals, and other bioactive compounds present in the diet.
                    - Microbial metabolites : include short-chain fatty acids (SCFAs) produced by gut bacteria through the fermentation of dietary fiber, as well as other metabolites with diverse biological activities.
                    - Biofluid : such as blood, urine, cerebrospinal fluid, and saliva. These metabolites can provide valuable information about the metabolic status of an individual or the effects of certain conditions or interventions. Biofluid metabolites can be derived from endogenous metabolism, dietary intake, or the metabolic activities of microbes within the body.

NMR and data acquisition
    Nuclear Magnetic Resonance (NMR) is a powerful analytical technique used to study the structure, dynamics, and interactions of molecules. It is based on the principle that atomic nuclei with an odd number of protons or neutrons have a magnetic moment and can be manipulated by an external magnetic field.

In NMR experiments, a sample is placed in a strong and static magnetic field, typically generated by a superconducting magnet. The static magnetic field aligns the nuclear spins of the sample along the field direction. Radiofrequency (RF) pulses are then applied to perturb the spin alignment, causing the nuclei to absorb and emit energy at specific frequencies.

The data acquisition process in NMR involves several steps:

1. Sample Preparation: The sample to be analyzed is prepared in a suitable solvent and placed in a specialized NMR tube or sample holder. The sample should be homogeneous and free from impurities that may interfere with the NMR signals.

2. Shim and Lock: Before data acquisition, the magnetic field homogeneity is optimized using shimming techniques. This ensures that the static magnetic field is as uniform as possible throughout the sample volume. Additionally, a lock system is employed to maintain a stable magnetic field during the experiment.

3. Pulse Sequence Selection: NMR experiments use specific pulse sequences tailored to the desired information about the sample. These pulse sequences consist of a series of RF pulses and delays that manipulate the nuclear spins and generate the desired NMR signals.

4. Excitation and Relaxation: The RF pulse excites the nuclear spins in the sample, causing them to precess around the magnetic field. After the excitation, the spins gradually return to their equilibrium state through a process called relaxation, emitting signals that can be detected.

5. Signal Detection: NMR signals are typically detected using a coil or antenna called a radiofrequency (RF) probe. The probe detects the weak electromagnetic signals emitted by the sample as the nuclear spins relax. These signals are then amplified and processed for further analysis.

6. Fourier Transform: The raw NMR data, consisting of time-domain signals, are transformed into frequency-domain spectra using a mathematical technique called Fourier transform. The resulting spectrum represents the frequencies and intensities of the NMR signals, providing information about the chemical shifts and coupling patterns of the sample's nuclei.

7. Data Processing and Analysis: The acquired NMR spectra are processed and analyzed using specialized software. This may involve baseline correction, phase correction, peak picking, integration, and further manipulation to extract useful information about the sample's chemical structure, molecular dynamics, and interactions.

workflow of metabolic profiling
Experimental design : This step involves planning the study objectives, selecting appropriate sample types, defining the sample size, and determining the experimental conditions. The experimental design should be carefully considered to ensure it aligns with the research goals and allows for meaningful analysis.
-----> Sample collection : Samples relevant to the study, such as blood, urine, tissue, or cells, are collected from the subjects or experimental systems. It is essential to follow proper collection protocols to minimize potential biases and ensure sample integrity.
-----> Sample preparation and analysis : The collected samples undergo various procedures to extract the metabolites of interest. This may involve processes such as extraction, purification, derivatization, or enzymatic reactions, depending on the sample type and the analytical techniques employed.
-----> NMR data preprocessing : For NMR-based metabolic profiling, the acquired raw NMR data undergo several preprocessing steps. These include baseline correction, phasing, referencing, and spectral alignment to remove systematic variations and enhance the quality of the spectra.
-----> Data Analysis & Integration/Network : In this step, statistical and multivariate analysis methods are applied to the preprocessed NMR data to identify significant metabolite variations across different sample groups or conditions. Multivariate techniques such as principal component analysis (PCA) or partial least squares-discriminant analysis (PLS-DA) are often used to explore patterns and relationships within the data. Integration with other omics data (e.g., genomics, proteomics) or network analysis can provide a more comprehensive understanding of the metabolic pathways and biological systems under investigation.
-----> Metabolite identification : After identifying statistically significant metabolite variations, the next step is to assign identities to the metabolites. This can be achieved by comparing the experimental NMR data with spectral databases or using advanced techniques such as two-dimensional NMR spectroscopy or mass spectrometry (MS) for structural elucidation.
-----> Interpretation & Validation : Once metabolites are identified, their biological relevance and potential implications are interpreted in the context of the study objectives. This step involves comparing the findings with existing knowledge, biological pathway analysis, and validation through additional experiments or replication studies.

Applications of NMR metabolomics
1.Diagnostics - NMR metabolomics can be used in clinical diagnostics to analyze biofluids such as urine, plasma, and feces for the identification of metabolic biomarkers associated with various diseases. For example, it can be applied to detect and monitor biomarkers related to periductal fibrosis, a condition characterized by abnormal tissue growth around the bile ducts.
2.Protocol development - NMR metabolomics can aid in the development of protocols for sample collection and preparation for specific applications. For instance, in the case of fecal sample collection, NMR can be used to optimize the sample handling and extraction methods to obtain high-quality metabolic profiles.
3.Bioactive compounds - NMR metabolomics can contribute to the discovery and characterization of bioactive compounds, including those derived from plants with potential anti-cancer properties. By analyzing the metabolic profiles of plant extracts or fractions, NMR helps identify specific metabolites that exhibit bioactivity and can be further investigated for drug development.
4.Personalised nutrition & animal feed - NMR metabolomics can be utilized to study the metabolic profiles of individuals or animal populations to understand their nutritional requirements and optimize personalized nutrition or animal feed formulations. For example, the metabolite profile of black soldier fly (BSF) can be analyzed using NMR to develop novel animal feed formulations.
5.Plant growth & harvest - NMR metabolomics can be applied to assess the metabolic changes in plants during different growth stages and harvesting conditions. By analyzing age-associated metabolic fingerprints, NMR helps understand the physiological changes that occur in plants over time and optimize growth conditions for improved crop yield and quality.
6.Gut health - NMR metabolomics is valuable in studying gut health, including the effects of bariatric surgery and the role of the gut microbiome. It can help identify metabolic markers associated with changes in gut microbiota composition and assess the impact of interventions on gut health.

coding
path = system.file("extdata/", package = 'nmrdata'); print(path)
read1d_proc(path, exp_type = list(PULPROG='noesypr1d'))
spec(x[15,], ppm, shift = range(ppm), interactive = T)
matspec(x, ppm, shift = range(ppm), interactive = T)
meta$run0der=rank(as.POSIXct(meta$a_DATE))
specOverlay(x, ppm, shift = c(-0.05, 0.05), an=list('Facet'=meta$a_EXP,#facet 'Run Order'=meta$runOder,#color 'Pulse Program' = meta$a_PULPROG), #line type
)
